neo/Ledger/Blockchain.cs

```diff
@@ -218,11 +217,32 @@ public static UInt160 GetConsensusAddress(ECPoint[] validators)
             return Contract.CreateMultiSigRedeemScript(validators.Length - (validators.Length - 1) / 3, validators).ToScriptHash();
         }
 
-        public IEnumerable<Transaction> GetMemoryPool()
+        public IEnumerable<Transaction> GetMemoryPoolVerified()
+        {
+            return mem_pool.GetVerifiedTransactions();
+        }
+
+        public IEnumerable<Transaction> GetMemoryPoolVerifiedAndSorted()
+        {
+            return mem_pool.GetSortedVerifiedTransactions();
+        }
+
+        public IEnumerable<Transaction> GetMemoryPoolVerifiedAndUnverified()
         {
             return mem_pool;
         }
 
+        public void GetMemoryPoolSeparateVerfiedAndUnverified(out IEnumerable<Transaction> verifiedTransactions,
```

Comment:
Maybe `GetMemoryPoolSeparateVerfiedAndUnverified` to GetMemoryPoolVerfiedAndUnverifiedPair or GetMemoryPoolPairVerfiedAndUnverified